content/blog/2024-08-23-roadmap-may-2024-retro.md

```diff
@@ -0,0 +1,245 @@
+
+- **Priority**: High
+- **Status**: [xilem#384](https://github.com/linebender/xilem/pull/384) implements most of this, the rest is deferred.
+  There are known issues with the size of the log files created by default in very large apps.
+
+### Fix glaring documentation problems [(original)](@/blog/2024-06-17-roadmap-may-2024.md#fix-glaring-documentation-problems)
+
+- **Priority**: High
+- **Status**: The issues mentioned in this item were addressed in [xilem#329](https://github.com/linebender/xilem/pull/329) and [xilem#503](https://github.com/linebender/xilem/pull/503).
+
+### Finish repository port [(original)](@/blog/2024-06-17-roadmap-may-2024.md#fix-glaring-documentation-problems)
+
+- **Priority**: High
+- **Status**: This was completed.
+  We moved all Masonry issues into the Xilem repository, and updated all references in code to use full links.
+  This move was coordinated in [#linebender > Moving Masonry into Xilem](https://xi.zulipchat.com/#narrow/stream/419691-linebender/topic/Moving.20Masonry.20into.20Xilem).
+
+### Write “widgets in arena” RFC [(original)](@/blog/2024-06-17-roadmap-may-2024.md#write-widgets-in-arena-rfc)
+
+- **Priority**: High
+- **Status**: The RFC was [rfcs#6](https://github.com/linebender/rfcs/pull/6), which was mostly implemented in [xilem#396](https://github.com/linebender/xilem/pull/396).
+
+### Write “pass order” RFC [(original)](@/blog/2024-06-17-roadmap-may-2024.md#write-pass-order-rfc)
+
+- **Prioriy**: High
+- **Status**: [rfcs#7](https://github.com/linebender/rfcs/pull/7) is in draft, and several important parts of the RFC have been completed.
+
+### Rewrite documentation [(original)](@/blog/2024-06-17-roadmap-may-2024.md#rewrite-documentation)
+
+- **Priority**: Medium
+- **Status**: This work has been deferred until the "pass order" refactor is complete, as we expect Masonry's core to be quite stable at that point.
+
+### Improve focus handling [(original)](@/blog/2024-06-17-roadmap-may-2024.md#improve-focus-handling) ?
+
+- **Priority**: Medium
+- **Status**: Not done.
+
+### Improve pointer status handling [(original)](@/blog/2024-06-17-roadmap-may-2024.md#improve-pointer-status-handling) ?
+
+- **Priority**: Medium
+- **Status**: Basic translation of touch events was added in ([xilem#313](https://github.com/linebender/xilem/pull/313)).
+  This unblocked using Masonry on devices where pointer input is not expected (primarily on Android).
+  We have split the remainder of this task into two tasks, with a priority placed on completing the parts which unblock other work (such as scrolling on Android).
+
+### Progress beyond roadmap {#masonry-extras}
+
+Masonry is in a state of flux, however is starting to see a few exploratory users.
+To unblock these users, some additional important features were added outside of the roadmap, such as:
+
+- Support for embedding Masonry inside a pre-existing event loop ([xilem#417](https://github.com/linebender/xilem/pull/417)).
```

Comment:
The link is a 404. I *think* you're referring to [xilem#325](https://github.com/linebender/xilem/pull/325)?

Reply:
No, that link is correct

GitHub has deleted the pull request, but the change I'm referring to happened in that PR.

Reply:
I've linked to https://github.com/linebender/xilem/commit/cf3530097baf0fc5c458593e215a19973572b991 instead.